### PR TITLE
Fix warnings making  heroku app to fail on deployement

### PR DESCRIPTION
**Pull request by felixkiryowa**
#### What does this PR do?

-  Resolving warning that is making Heroku building  the application fail

#### Description of Task to be completed?
- Removing unused variables, indicated by eslint which make the Heroku build to fail

#### What are the relevant pivotal tracker stories?
[#165879303](https://www.pivotaltracker.com/story/show/165879303)

[Finishes #165820365]

### CODE REVIEW

File: src/components/InputBox/index.js [hunk 1]
```diff
@@ -3,12 +3,18 @@ import React from 'react';
...
+		<div className='inputContain'>
+			<input
+				type={type}
+				value={value}
+				name={name}
+				placeholder={placeholder}
+				className='input'
+				onChange={(val) => onChangeInput(val)}
+			/>
+		</div>
```
Comment (ezrogha) [inline src/components/InputBox/index.js]: @felixkiryowa please work on this indentation,
It should be two spaces

**Review by ezrogha — CHANGES_REQUESTED** [review]
Well done with this fix @felixkiryowa,
Kindly work on all indentations, they should be two spaces

**Comment by adesege** [comment]
@felixkiryowa Fix the merge conflict on this PR

**Comment by felixkiryowa** [comment]
I have closed this PR because , Heroku was fixed and it is working

**Pull request closed without merging**